content/advent-2019/cmdline.md

```diff
@@ -0,0 +1,325 @@
++++
+title = "Writing Friendly Command Line Applications"
+date = "2019-12-08T00:00:00+00:00"
```

Comment:
Please change the date to 2019-12-02 for publishing tomorrow

Reply:
Done